.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "espressif-training",
+  // Choose between "image" and "build" properties to pull built image or build from Dockerfile
+  "image": "docker.io/sergiogasquez/esp-rs-env:esp32c3-training",
```

Comment:
I suggest to use the `build` property with the local `Dockerfile` as the default, otherwise the dev environment relies on this particular image to be available.